mindjunkies/forums/tests/test_models.py

```
@@ -3,125 +1,204 @@
+            content='Second comment',
+            author=self.user
         )
+        comments = ForumComment.objects.all()
+        self.assertEqual(comments[0], self.comment)  # Older comment first
 
-        last_activity = max(
-            last_comment.created_at if last_comment else None,
-            last_reply.created if last_reply else None,
+    def test_reply_ordering(self):
+        """Test Reply Meta ordering"""
+        reply2 = Reply.objects.create(
```

Ruff (0.8.2)

Local variable `reply2` is assigned to but never used

Remove assignment to unused variable `reply2`

(F841)